workflow/defaults/default_config.yaml

```diff
@@ -32,6 +36,6 @@ Annotations:
     logDefault: -1.0        # default
     repairMissing: False    # default
 Cytoscape: 
-    path: ~\Documents\Comp_bio\Cytoscape_v3.7.1 
+    path: C:\Users\ajshe\OneDrive\Documents\Comp_bio\Cytoscape_v3.7.1
```

Comment:
Let's not use a specific path here.  It should be generic so a user can run TPS with this default file.  Should we make Cytoscape True by default?

Reply:
What would be the best approach to using paths that are not relative to the current working directory? This default behavior assumes that the Cytoscape application lives in a different location somewhere on the file system that is independent of TPS

Reply:
We discussed changing Docker: Cytoscape: default to True so that this path isn't used.  Then `path` here can be a placeholder like `Cytoscape_v3.7.1` and no path.